artifactory/commands/commandssummaries/buildinfosummary.go

```diff
@@ -46,6 +59,23 @@ func (bis *BuildInfoSummary) buildInfoTable(builds []*buildInfo.BuildInfo) strin
 	return tableBuilder.String()
 }
 
+func (bis *BuildInfoSummary) buildInfoModules(builds []*buildInfo.BuildInfo) string {
```

Comment:
Let's add a unit test for this function, validating the output, including the case of stripping unsupported module types.